tests/components/advantage_air/test_sensor.py

```diff
@@ -42,6 +42,15 @@ async def test_sensor_platform(hass, aioclient_mock):
 
     # Test First TimeToOn Sensor
     entity_id = "sensor.ac_one_time_to_on"
+
+    registry.async_update_entity(entity_id=entity_id, disabled_by=None)
+    await hass.async_block_till_done()
+    async_fire_time_changed(
+        hass,
+        dt.utcnow() + timedelta(seconds=RELOAD_AFTER_UPDATE_DELAY + 1),
+    )
+    await hass.async_block_till_done()
+
```

Comment:
If I understand this correctly, this block (and the identical one below) is meant to enable the entity since this PR changes it to disabled by default, with the time change meant to trigger the built-in reload after entity update. It took me about 10 minutes of searching to figure all of that out, so it might be worth factoring these out into a helper method `_enable_entity` and putting a comment as to the purpose of the time change.

Reply:
Yeah I dont love this either, but apparently its the right way to deal with disabled entities. If I wasnt going to remove these in the near future a refactor would be good.